src/main/drivers/pwm_output.c

```diff
@@ -255,7 +256,8 @@ void motorInit(const motorConfig_t *motorConfig, uint16_t idlePulse, uint8_t mot
 
 #ifdef USE_DSHOT
         if (isDigital) {
-            pwmDigitalMotorHardwareConfig(timerHardware, motorIndex, motorConfig->motorPwmProtocol);
+            pwmDigitalMotorHardwareConfig(timerHardware, motorIndex, motorConfig->motorPwmProtocol,
```

Comment:
Not sure here; maybe polarity shall be handled internally in `pwmDigitalMotorHardwareConfig`